rpc/receiveranswer_test.go

```diff
@@ -81,7 +79,7 @@ func TestBootstrapReceiverAnswerRpc(t *testing.T) {
 
 	res, rel := c.Call(ctx, func(p testcapnp.CapArgsTest_call_Params) error {
 		capId := p.Message().AddCap(c.Client.AddRef())
-		p.SetCap(capnp.NewInterface(p.Segment(), capId).ToPtr())
+		p.SetCap(capnp.NewInterface(p.Segment(), capId).Client())
```

Comment:
Again, I think you can skip the AddCap/NewInterface dance and just do `SetCap(c.Client.AddRef())`.

Reply:
Fixed.